packages/block-library/src/rss/index.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * Server-side rendering of the `core/rss` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the `core/rss` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the block content with received rss items.
+ */
+function render_block_core_rss( $attributes ) {
+
+	if ( ! isset( $attributes['feedURL'] ) ) {
+		return;
+	}
+
+	$rss = fetch_feed( $attributes['feedURL'] );
+
+	if ( is_wp_error( $rss ) ) {
+		return '<div class="components-placeholder"><div class="notice notice-error"><strong>' . __( 'RSS Error:', 'gutenberg' ) . '</strong> ' . $rss->get_error_message() . '</div></div>';
+	}
+
+	if ( ! $rss->get_item_quantity() ) {
+		$rss->__destruct();
+		unset( $rss );
+		return '<div class="components-placeholder"><div class="notice notice-error">' . __( 'An error has occurred, which probably means the feed is down. Try again later.', 'gutenberg' ) . '</div></div>';
+	}
+
+	$items = (int) $attributes['itemsToShow'];
+	if ( $items < 1 || 10 < $items ) {
+		$items = 5;
+	}
+
+	$list_items = '';
+	foreach ( $rss->get_items( 0, $items ) as $item ) {
+		$title = esc_html( trim( strip_tags( $item->get_title() ) ) );
+		if ( empty( $title ) ) {
+			$title = __( '(Untitled)', 'gutenberg' );
+		}
+
+		$link = $item->get_link();
+		$link = esc_url( strip_tags( $link ) );
```

Comment:
Why the `strip_tags`? Shouldn't `esc_url` be enough already?

Reply:
Should be enough. This line is from the rss widget.